mapie/regression/regression.py

```diff
@@ -639,16 +643,15 @@ def predict(
                 check_alpha_and_n_samples(alpha_np, n)
 
             # Predict the target with confidence intervals
-            y_pred, y_pred_low, y_pred_up = \
-                self.conformity_score_function_.get_bounds(
-                    X,
-                    self.estimator_,
-                    self.conformity_scores_,
-                    alpha_np,
-                    ensemble=ensemble,
-                    method=self.method,
-                    optimize_beta=optimize_beta,
-                    allow_infinite_bounds=allow_infinite_bounds
-                )
+            outputs = self.conformity_score_function_.predict_set(
```

Comment:
predict_set function is not required, why not just call get_bounds here ?

Reply:
After careful consideration, we're going to go with the first proposal.

Reply:
-> we agree that this should not be done in this PR